tests/benchmarks/fio_benchmark_test.py

```diff
@@ -153,5 +154,62 @@ def TestZeroMinutes(self):
     self.assertEquals(proc.call_count, 0)
 
 
+class TestFioTargetModeFlag(unittest.TestCase):
+  def doTargetModeTest(self, mode, fill_device=None, against_device=None,
+                       format_disk=None):
+    with mock.patch(fio_benchmark.__name__ + '.FillDevice') as fill_device, \
```

Comment:
Consider renaming "fill_device" here to FillDevice (retain its original method name) so the reader knows this refers to a method/function rather than a local variable.

Similar comment to all the names below that are extracted from the fio_benchmark module. ("get_job_string", etc)